### ISSUE TITLE
[Bug] Accessibility - Enforce manual page zoom does not work/survive app restart

### ISSUE THREAD

**Issue by smileatom** [duckduckgo/Android#1902]
### Describe the bug

Hi,

I cant use mobile browsers that dont allow force zoom override since many websites lock down very small fonts and I cant read them.

Sadly, the implementation of Accessibility/Enforce manual page zoom does not work. If you enable it while a webpage is open it will mostly work if you are lucky. However it will not work after a restart of the DDG app even though the setting is still enabled in settings.

This affects android.

I will use chrome until this bug is fixed.

A good test site is https://techmeme.com

### How to Reproduce

1)  Go to https://techmeme.com   try to manually zoom in the webpage.
2) Go to DDG settings/accessibility/Enforce manual page zoom and toggle it on
3) Try to manually zoom in techmeme.com it may or may not allow you (But usually doesnt)
4) Refresh the page and try the manual zoom again it may or may not allow you but usually doesnt.
5) Kill the DDG app and open DDG again to techmeme.com try to manually zoom the page - it wont allow you
6) Go back into settings/accessibility/Enforce manual page zoom to visually confirm that the toggle is still enabled.
7) Fix bug

### Expected behavior

I should be able to zoom pages that dont allow page zoom on mobile.

### Environment

```markdown
- DDG App Version: 5.123.0
- Device:  Galaxy S22 Ultra
- OS:  Android
```

**Comment by smileatom** [comment]
Still broken

**Comment by cmonfortep** [comment]
Sorry about that. Techmeme scenario is a bit weird. It's true it doesn't work first time you load the site, but it works when you navigate within that site, even clicking on a different tab fixes the issue. Most probably it's a timing issue between site loading and our JS code to override viewport metadata.

That being said, the setting works even after restarting the app. If you have more sites where this doesn't work, please share them.

Thanks.

**Comment by smileatom** [comment]
> Sorry about that. Techmeme scenario is a bit weird. It's true it doesn't work first time you load the site, but it works when you navigate within that site, even clicking on a different tab fixes the issue. Most probably it's a timing issue between site loading and our JS code to override viewport metadata.
> 
> That being said, the setting works even after restarting the app. If you have more sites where this doesn't work, please share them.
> 
> Thanks.

So I assumed DDG was based on chrome, both chrome and the Samsung android fork of chrome seem to work fine.  I have encountered other sites that dont work.  Even when I have been able to get it to work it stops working on restart for the pages it doesnt work on. Sorry I dont have time to find more issues like this, if its a timing issue with javascript hooks its probably the wrong approach to solving that problem.
